### PR TITLE
Editorial tweaks

### PR DESCRIPTION
**Pull request by gibson042**
* Fix some typos.
* Change some phrasing for readability.
* Merge sections describing expectations of attributes and properties vs. methods.
* Update a section title to imperative form.

***
<a href="https://pr-preview.s3.amazonaws.com/gibson042/design-principles/pull/409.html" title="Last updated on Dec 24, 2022, 5:29 PM UTC (bde8146)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3ctag/design-principles/409/69b2ff0...gibson042:bde8146.html" title="Last updated on Dec 24, 2022, 5:29 PM UTC (bde8146)">Diff</a>

### CODE REVIEW

File: index.bs
Comment (LeaVerou) [inline index.bs:1694]: What happened to this section?

**Review by LeaVerou — CHANGES_REQUESTED** [review]
See comment about missing section but LGTM otherwise! Thank you!

File: index.bs
Comment (gibson042) [reply index.bs:1694]: Merged into #attributes-like-data along with #attributes-vs-methods, eliminating the redundancy of three sections describing appropriate behavior for attributes/properties vs. methods and lamenting their violation by `offsetTop`.

File: index.bs
Comment (LeaVerou) [inline index.bs:2537]: I would use an `<abbr>` here

File: index.bs
Comment (LeaVerou) [reply index.bs:1694]: I agree there is redundancy, and thank you for spotting it! However, right now it feels like a lot of the more concrete guidance was dropped in the process (e.g. the parts about what should equal what). Could you try to fold some of that in?

Comment (gibson042) [reply index.bs:1694]: I believe all the points are still there.
* "_Getters must not have any observable side effects._" is currently in all three sections and remains in the proposed merge.
* "_Getters should not perform any {complex,blocking,[inefficient]} operations._" is currently in all three sections and remains in the proposed merge.
* A variation of "_Ensure that `obj.attribute === obj.attribute` is always true. Don't create a new value each time the getter is called._" is currently in all three sections remains in the proposed merge with the least passive phrasing.
* "_[ensure that] given `obj.attribute = x`, [a subsequent] `obj.attribute === x` {is,should be} true._" is currently in all three sections and remains in the proposed merge. And I have now also incorporated the elaborations currently absent from [#attributes-like-data](https://w3ctag.github.io/design-principles/#attributes-like-data).
* Guidance that static objects should be new each time (and therefore returned from methods rather than attributes) is currently absent from [#attributes-like-data](https://w3ctag.github.io/design-principles/#attributes-like-data) but is incorporated in the proposed merge.
* Guidance to use a method if any of the stated prescriptions cannot be satisfied is currently in all three sections and remains in the proposed merge (as the first content after the list of them).
* An observation that `offsetTop` violates the second point and should have been a method is currently in all three sections and remains in the proposed merge (in its clearest form and with outbound links).

File: index.bs
Comment (gibson042) [reply index.bs]: Done, and also for IANA.

File: index.bs
Comment (LeaVerou) [reply index.bs:1694]: @cynthia what do you think?

File: index.bs
Comment (LeaVerou) [reply index.bs]: Thanks!

**Comment by LeaVerou** [comment]
Hi @gibson042, we were just reviewing this in a breakout in our Tokyo F2F but noticed it needs rebasing. Could you please rebase it? Thanks!

**Comment by cynthia** [comment]
Please hold off on the rebase. We discussed this during our F2F and concluded this changes too much at once. For that reason, we don't think we can take this whole PR as is. I'll slowly incorporate some of the pure editorial changes and typos.

**Comment by cynthia** [comment]
Apologies for the long delay, and we appreciate the contribution - but I think it's more work to resolve the conflicts and land this at this point. We'll incorporate the suggestions in a different patch and attribute it accordingly, but likely in a more conservative form. Will follow up in a week or two with a revised PR incorporating this.

**Pull request closed without merging**
